wp-content/themes/pawar2018/sass/modules/_header.scss

```diff
@@ -77,18 +79,47 @@
   width: 20px;
   height: 16px;
   cursor: pointer;
-
-  &:after {
-    content: '';
-    position: absolute;
-    top: 0;
-    left: 0;
+  margin: 0 .75rem;
+  .line {
+    background-color: white;
+    border-radius: 4px;
     display: block;
-    width: 100%;
     height: 2px;
-    background: #fefefe;
-    box-shadow: 0 7px 0 #fefefe,
-    0 14px 0 #fefefe;
+    left:0;
+    position: absolute;
+    -webkit-transition: all 320ms ease-in;
+    transition: all 320ms ease-in;
+    width: 100%;
+  }
+  .line-top {
+    top:0;
+  }
+  .line-middle {
+    top: 50%;
+  }
+  .line-bottom {
+    top:100%;
+  }
+  &:hover .line-top,
+  &:focus .line-top {
+    -webkit-transform: translateY(-2px);
+            transform: translateY(-2px);
+  }
+  &:hover .line-bottom,
+  &:focus .line-bottom {
+    -webkit-transform: translateY(2px);
+            transform: translateY(2px);
+  }
+  &.header-burger--active .line-top {
+    -webkit-transform: translateY(8px) translateX(0) rotate(225deg);
+            transform: translateY(8px) translateX(0) rotate(225deg);
+  }
+  &.header-burger--active .line-middle {
+    opacity: 0;
+  }
+  &.header-burger--active .line-bottom {
+    -webkit-transform: translateY(-8px) translateX(0) rotate(-225deg);
+            transform: translateY(-8px) translateX(0) rotate(-225deg);
```

Comment:
Nit-pick - since we're using ye olde Sass, might as well take advantage of nesting to avoid typing `&.header-burger--active` three times. Nest these in a single parent selector?